src/Data/Align/Text.hs

```diff
@@ -0,0 +1,60 @@
+{-# language RecordWildCards #-}
```

Comment:
If you have even anecdotal knowledge, please add a few lines of comment about expectable gains vs using stringy version.

Reply:
I think better still would be to set up a benchmark suite.